src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -3396,6 +3396,8 @@ private int MinBatchSize
         }
 
         private DateTime _lastCheckedFlushLock;
+        
+        public int CreatedFieldsCount;
```

Comment:
I think this should be a part of CurrentIndexingScope it is created per batch?

Reply:
No, this is per document, not per batch.
This is needed here because I need to reset it in the right place and this is the simplest location to track this.